app/src/main/res/layout/fragment_pledge.xml

```diff
@@ -0,0 +1,182 @@
+          android:orientation="vertical">
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/Kickstarter_is_not_a_store" />
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/Its_a_way_to_bring_creative_projects_to_life" />
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/Learn_more_about_accountability"
+            android:textColor="@color/primary" />
+        </LinearLayout>
+
+      </LinearLayout>
+
+      <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingEnd="@dimen/activity_vertical_margin"
+        android:paddingStart="@dimen/activity_vertical_margin">
+
+        <include layout="@layout/horizontal_line_1dp_view" />
+
+        <TextView
+          style="@style/CalloutPrimaryMedium"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="@dimen/grid_4"
+          android:text="@string/Your_pledge_amount" />
+
+        <TextView
+          android:id="@+id/pledge_amount"
+          style="@style/PledgeCurrencySecondary"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          tools:text="$20" />
+
+        <TextView
+          style="@style/CalloutPrimaryMedium"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:text="@string/Your_shipping_destination" />
+
+        <TextView
+          android:id="@+id/shipping_amount"
+          style="@style/PledgeCurrency"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="end"
+          android:layout_marginBottom="@dimen/grid_4"
+          tools:text="$20" />
+
+        <include layout="@layout/horizontal_line_1dp_view" />
+
+        <TextView
+          style="@style/CalloutPrimaryMedium"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/grid_1"
+          android:layout_marginTop="@dimen/grid_4"
+          android:text="@string/dashboard_graphs_funding_total" />
+
+        <LinearLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/grid_4"
+          android:orientation="horizontal">
+
+          <TextView
+            style="@style/FootnoteSecondary"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
```

Comment:
Shouldn't this copy always be present? 
<img src=https://user-images.githubusercontent.com/16387538/55734064-e889a700-59ec-11e9-994e-ad94eeca56f4.jpg width=200/> <img src=https://user-images.githubusercontent.com/16387538/55734302-4e762e80-59ed-11e9-9881-11ac3053abcd.png width=200/>

Reply:
It has a weight